packages/web-integration-oc10/lib/Application.php

```diff
@@ -33,4 +34,8 @@ class Application extends App {
 	public function __construct(array $urlParams = []) {
 		parent::__construct(Application::APPID, $urlParams);
 	}
+
+    public function boot(): void {
+        Util::addscript(Application::APPID, 'app');
```

Comment:
```suggestion
        Util::addScript(Application::APPID, 'app');
```
don't know if that's required, but the function name is `addScript` with an uppercase `S`.